util/chplenv/chpl_jemalloc.py

```diff
@@ -1,26 +1,55 @@
 #!/usr/bin/env python3
 import os
 import sys
+import optparse
 
-import chpl_mem, overrides, third_party_utils
+import chpl_mem, chpl_platform, overrides, third_party_utils
 from utils import error, memoize, run_command, warning
 
 
 @memoize
-def get():
-    jemalloc_val = overrides.get('CHPL_JEMALLOC')
-    mem_val = chpl_mem.get('target')
-    if not jemalloc_val:
-        if mem_val == 'jemalloc':
+def get(target='target'):
```

Comment:
Using the arg name of `target` here feels a little confusing to me since target=host feels slightly off. In other chplenv scripts I think we just call the name `flag`, which isn't as descriptive but doesn't have the double use of target.